blocks/library/image/block.js

```diff
@@ -190,9 +201,7 @@ class ImageBlock extends Component {
 			controls,
 			focus && (
 				<InspectorControls key="inspector">
-					<BlockDescription>
-						<p>{ __( 'Worth a thousand words.' ) }</p>
-					</BlockDescription>
+					{blockDescription}
```

Comment:
Minor: Changing {blockDescription} to { blockDescription } make the change more in line with the codebase. The Gallery block currently misses the spaces and will be corrected.